couchbase/behavior_options.hxx

```diff
@@ -60,19 +60,10 @@ public:
     return *this;
   }
 
-  /**
-   * Selects network to use.
-   *
-   * @param name network name as it is exposed in the configuration.
-   * @return this object for chaining purposes.
-   *
-   * @see
-   * https://docs.couchbase.com/server/current/learn/clusters-and-availability/connectivity.html#alternate-addresses
-   * @see https://docs.couchbase.com/server/current/rest-api/rest-set-up-alternate-address.html
-   */
+  [[deprecated("Use couchbase::network_options#preferred_network() instead.")]]
   auto network(std::string name) -> behavior_options&
   {
-    network_ = std::move(name);
+    [[maybe_unused]] auto name_ = std::move(name);
```

Comment:
Even though it's deprecated, shouldn't we still use this if it's set? (but give priority to `network_options.network` if both of them are set)

Reply:
I think, you right. I will adjust the change